### ISSUE TITLE
swift package manager build failure while using CommandLine

### ISSUE THREAD

**Issue by btbytes** [jatoben/CommandLine#98]
This error occurs when `swift build` is run:

```
error: the directory Tests/CommandLine has an invalid name ('CommandLine'): the name of a test module has no 'Tests' suffix
fix: rename the directory 'Tests/CommandLine' to have a 'Tests' suffix
```

The package dependency is defined as follows:
```
╭─pradeep@home ~/csv2md ‹system› 
╰─➤     cat Package.swift                                                                                                                                                                                    1 ↵
// swift-tools-version:3.1

import PackageDescription

let package = Package(
    name: "csv2md",
    dependencies: [
        .Package(url: "https://github.com/Daniel1of1/CSwiftV", majorVersion: 0),
        .Package(url: "https://github.com/jatoben/CommandLine", majorVersion: 2, minor: 2),
        .Package(url: "https://github.com/onevcat/Rainbow", majorVersion: 2)
    ]
)
```

Me being a newbie to Swift, I might be missing something obvious!

**Comment by matejkosiarcik** [comment]
This was already fixed on ```master``` (See [#78](https://github.com/jatoben/CommandLine/pull/78)).
Although it is not in stable release yet, you can *temporary* depend on pre-release to fix it:
```.Package(url: "https://github.com/jatoben/CommandLine", "3.0.0-pre1"),```

**Comment by btbytes** [comment]
Thanks! this works.
